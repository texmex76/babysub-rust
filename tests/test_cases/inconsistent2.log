c BabySub Subsumption Preprocessor
c reading from '/home/gstrein/Documents/repos/babysub-rust/tests/test_cases/inconsistent2.cnf'
c parsed 'p cnf 4 4' header
c checked:                      0    clauses 1.00 per subsumed
c subsumed:                     3    clauses 75%
c process-time:                 0.00 seconds
