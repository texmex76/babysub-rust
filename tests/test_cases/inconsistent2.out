c BabySub Subsumption Preprocessor
c reading from '/home/gstrein/Documents/repos/babysub-rust/tests/test_cases/inconsistent2.golden'
c parsed 'p cnf 4 1' header
c hash-signature: 71876167
p cnf 4 1
 0
c checked:                      0    clauses 0.00 per subsumed
c subsumed:                     0    clauses 0%
c process-time:                 0.00 seconds
